src/index.html

```diff
@@ -11,7 +11,191 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>
+          Personal information
+        </legend>
+
+        <label for="username">
+          Surname:
+          <input
+            class="input-field"
+            autocomplete="off"
+            type="text"
+            name="surname"
+            id="username"
+          >
+        </label>
+
+        <label for="name" class="label">
+          Name:
+          <input
+            class="input-field"
+            autocomplete="off"
+            type="text"
+            name="surname"
+            id="name"
+          >
+        </label>
+
+        <label for="tentacles" class="label">
+          How old are You?
+          <input
+            class="input-field"
+            type="number"
+            name="tentacles"
+            id="tentacles"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </label>
+
+        <label for="fullDate" class="label">
+          Full date of birth:
+          <input
+            class="input-field"
+            type="date"
+            name="trip-start"
+            id="fullDate"
+          >
+        </label>
+
+        <label for="policy">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="policy-agree"
+            id="policy"
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+
+        <label for="email" class="label">
+          E-mail:
+          <input
+            class="input-field"
+            autocomplete="off"
+            required
+            placeholder="email@example.com"
+            type="email"
+            name="email"
+            id="email"
+          >
+        </label>
+
+        <label
+          for="password"
+          class="label"
+          required
+        >
+          Password:
+          <input
+            required
+            autocomplete="off"
+            type="password"
+            name="password"
+            id="password"
+            minlength="6"
+            maxlength="25"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="input-field">
+          <label for="chooseAnswer chooseAnswer2">
```

Comment:
chooseAnswer2 - what does 2 mean? 

```suggestion
          <label for="chooseAnswer chooseAnswer2">
```

Reply:
2 - second answer opinion